diff -Nur urt-3.1b.orig/get/getx11/XGetHClrs.c urt-3.1b/get/getx11/XGetHClrs.c
--- urt-3.1b.orig/get/getx11/XGetHClrs.c	Thu Apr 30 16:16:23 1992
+++ urt-3.1b/get/getx11/XGetHClrs.c	Fri Oct 12 22:21:44 2001
@@ -1,5 +1,4 @@
 #ifndef XLIBINT_H_NOT_AVAILABLE
-#include <X11/copyright.h>
 
 /* $XConsortium: XGetHClrs.c,v 11.10 88/09/06 16:07:50 martin Exp $ */
 /* Copyright    Massachusetts Institute of Technology    1986	*/
diff -Nur urt-3.1b.orig/lib/rle_global.c urt-3.1b/lib/rle_global.c
--- urt-3.1b.orig/lib/rle_global.c	Tue Jan 28 19:23:03 1992
+++ urt-3.1b/lib/rle_global.c	Fri Oct 12 22:21:44 2001
@@ -76,7 +76,7 @@
     8,				/* cmaplen (log2 of length of color map) */
     NULL,			/* pointer to color map */
     NULL,			/* pointer to comment strings */
-    stdout,			/* output file */
+    NULL,			/* output file */
     { 7 },			/* RGB channels only */
     0L,				/* Can't free name and file fields. */
     "Urt",			/* Default "program name". */
diff -Nur urt-3.1b.orig/lib/rle_hdr.c urt-3.1b/lib/rle_hdr.c
--- urt-3.1b.orig/lib/rle_hdr.c	Fri Jun 26 22:44:20 1992
+++ urt-3.1b/lib/rle_hdr.c	Fri Oct 12 22:21:44 2001
@@ -269,6 +269,9 @@
 {
     rle_hdr *ret_hdr;
 
+    if ( rle_dflt_hdr.rle_file == NULL)
+	rle_dflt_hdr.rle_file = stdout;
+
     if ( the_hdr == &rle_dflt_hdr )
 	return the_hdr;
 
diff -Nur urt-3.1b.orig/tools/into.c urt-3.1b/tools/into.c
--- urt-3.1b.orig/tools/into.c	Thu Apr 30 16:10:15 1992
+++ urt-3.1b/tools/into.c	Fri Oct 12 22:22:15 2001
@@ -40,10 +40,7 @@
 static char buf[MAXPATHLEN+1];
 short forceflg;				/* overwrite an unwritable file? */
 
-extern int errno;
-extern char *sys_errlist[];
-
-void
+int
 main(argc, argv)
 int argc;
 char **argv;
