diff -Nur urt-3.1b.orig/config/urt urt-3.1b/config/urt
--- urt-3.1b.orig/config/urt	Wed Jul 15 21:47:15 1992
+++ urt-3.1b/config/urt	Fri Oct 12 22:09:57 2001
@@ -19,7 +19,7 @@
 ##define BOBCAT         HP "bobcat" 9000 series (with Starbase).
 ##define CX3D           Chromatics CX3D system.
 ##define FB             BRL extended frame buffer library.
-##define GETICR		Display for NCSA telnet.  Requires PBMPLUS tools.
+#define GETICR		Display for NCSA telnet.  Requires PBMPLUS tools.
 ##define GRINNELL	Grinnell GMR27 frame buffer.
 ##define HPQCR		Matrix QCR-Z attached to HP series 300.
 ##define IRIS           Silicon Graphics Iris 2000/3000.
@@ -29,7 +29,7 @@
 ##define SUNTOOLS       For Suns running SunTools.
 ##define TAAC1		Suns with TAAC-1 board.
 ##define X10		X10 Window system.
-##define X11            X11 Window system.
+#define X11		X11 Window system.
 
 # Conversion programs
 #define ABEKASA60	Abekas A60 conversions.
@@ -39,11 +39,11 @@
 #define GRAYFILES	Grayscale conversions.
 ##define SGICONV	SGI Image conversions.
 #define MACPAINT	Macpaint conversions.
-##define PBMPLUS	PBMPLUS conversions.
+#define PBMPLUS		PBMPLUS conversions.
 #define POSTSCRIPT	Postscript conversions.
 ##define SUNRASTER	For Sun raster conversions.
 #define TARGA		Targa format conversions.
-##define TIFF		TIFF image conversions.
+#define TIFF		TIFF image conversions.
 #define TIFF2p4		Define if libtiff is version 2.4 or greater
 #define VICAR		VICAR image conversions.
 #define WASATCH		Wasatch Paintbox conversions.
@@ -60,7 +60,7 @@
 ##define APOLLOCC	Apollo computer.
 ##define CONVEXCC	Convex C 4.0 or greater
 ##define CRAY2CC		Cray-2 running UNICOS.
-##define GCC		Gcc compiler.
+#define GCC		Gcc compiler.
 ##define HPUX300CC	HP 9000/300 running HP-UX.
 ##define HPUX800CC	HP 9000/800 running HP-UX.
 ##define IBMR6000CC	IBM RS6000.
@@ -76,32 +76,32 @@
 # environment and desires.
 
 ##define ABEKAS_PAL	Convert to PAL format in Abekas A60 tool.
-##define ALL_MAN	Print/install all man pages.
-##define CONST_DECL	If your compiler understands 'const' (ANSI)
+#define ALL_MAN		Print/install all man pages.
+#define CONST_DECL	If your compiler understands 'const' (ANSI)
 ##define FILE_NAME_14_CHARS	If filenames are limited to 14 characters.
 ##define HAVE_CPROTO	If you have the cproto command.  Not normally required.
 ##define NEED_BSTRING	Your libc does not have bcopy or bzero (Sys V probably)
 ##define NEED_GETOPT	Your libc does not have the getopt function (BSD)
 ##define NEED_RENAME	Your libc does not have the rename function (Sys V)
 ##define NEED_SETLINEBUF	Your stdio does not have setlinebuf (Sys V)
-##define NO_MAKE_MAKEFILE	If you don't want a "makefile" action.
+#define NO_MAKE_MAKEFILE	If you don't want a "makefile" action.
 ##define NO_OPEN_PIPES	If file "names" "|..." and "*.Z" are not special.
 ##define NO_RANLIB	You don't have the ranlib program (Sys V)
 ##define NO_ITIMER	You don't have setitimer (e.g., Cray 2).
 ##define NO_TOOLS	Don't build tool directory.
 ##define STDIO_NEEDS_BINARY	Your fopen() needs "b" specified for binary io
-##define USE_TIME_H	You need to include time.h for time_t (Sys V)
-##define SYS_V_SETPGRP	Your system has the SysV setpgrp() call.
-##define USE_L_FLAG	Use -Llibdir -lrle instead of libdir/liblre.a.
-##define USE_PROTOTYPES	Declare library functions with ANSI-C prototypes.
-##define USE_RANDOM	Your libc has the BSD random() function.
-##define USE_STDARG	Use 'stdarg' instead of 'varargs' (ANSI).
-##define USE_STDLIB_H	You have "stdlib.h".
-##define USE_UNISTD_H	You have "unistd.h".
-##define USE_STRING_H	You have "string.h", not "strings.h"
-##define VOID_STAR	Your C compiler understands the 'void *' declaration.
-##define USE_XLIBINT_H	You have the include file <X11/Xlibint.h>.
-##define X_SHARED_MEMORY	You have the XShm extension.
+#define USE_TIME_H	You need to include time.h for time_t (Sys V)
+#define SYS_V_SETPGRP	Your system has the SysV setpgrp() call.
+#define USE_L_FLAG	Use -Llibdir -lrle instead of libdir/liblre.a.
+#define USE_PROTOTYPES	Declare library functions with ANSI-C prototypes.
+#define USE_RANDOM	Your libc has the BSD random() function.
+#define USE_STDARG	Use 'stdarg' instead of 'varargs' (ANSI).
+#define USE_STDLIB_H	You have "stdlib.h".
+#define USE_UNISTD_H	You have "unistd.h".
+#define USE_STRING_H	You have "string.h", not "strings.h"
+#define VOID_STAR	Your C compiler understands the 'void *' declaration.
+#define USE_XLIBINT_H	You have the include file <X11/Xlibint.h>.
+#define X_SHARED_MEMORY	You have the XShm extension.
 
 # DEST is where all programs will be put.  Can be overridden for specific
 # directories below.
@@ -191,11 +191,10 @@
 #ifdef PBMPLUS
 # PBMDIR should refer to the root of your PBM source tree.
 # If relative, it's relative to root of URT source tree.
-#defpath PBMDIR
+#defpath PBMDIR	/usr
 
-INCPBMPLUS = -I$(PBMDIR) -I$(PBMDIR)/ppm -I$(PBMDIR)/pgm -I$(PBMDIR)/pbm
-LIBPBMPLUS = $(PBMDIR)/ppm/libppm.a $(PBMDIR)/pgm/libpgm.a \
-		$(PBMDIR)/pbm/libpbm.a 
+INCPBMPLUS = -I$(PBMDIR)/include
+LIBPBMPLUS = -lppm -lpgm -lpbm
 #endif
 
 #ifdef SUNRASTER|SUNTOOLS|TAAC1
@@ -217,8 +216,8 @@
 #ifdef TIFF
 # Location of tiff library and include files.  Relative to URT root directory.
 #defpath TIFFDIR
-INCTIFF = -I$(TIFFDIR)
-LIBTIFF = $(TIFFDIR)/libtiff.a
+INCTIFF = -I/usr/include
+LIBTIFF = -ltiff
 #endif
 
 #ifdef X10
@@ -227,6 +226,6 @@
 #endif
 
 #ifdef X11
-LIBX11 = -lX11
-INCX11 = 
+LIBX11 = -L/usr/X11R6/lib -lX11
+INCX11 = -I/usr/X11R6/include
 #endif
diff -Nur urt-3.1b.orig/makefile.hdr urt-3.1b/makefile.hdr
--- urt-3.1b.orig/makefile.hdr	Fri Mar  6 15:04:57 1992
+++ urt-3.1b/makefile.hdr	Fri Oct 12 22:10:22 2001
@@ -45,7 +45,7 @@
 #emit MachFLAGS
 MachFLAGS =
 
-CFLAGS = -O $(DFLAGS) $(DIRFLAGS) $(ExtraCFLAGS) $(IFLAGS) $(MachFLAGS) 
+CFLAGS = $(DFLAGS) $(DIRFLAGS) $(ExtraCFLAGS) $(IFLAGS) $(MachFLAGS) 
 IFLAGS = -I. -I$(RI)
 
 #ifdef GCC
